### ISSUE TITLE
Undefined local variable or method config (v2.0.5)

### ISSUE THREAD

**Issue by TiuTalk** [pboling/sanitize_email#104]
Once we updated `sanitize_email` to `2.0.5`, our staging `assets:precompile` step started failing, down below are the logs with the relevant part.

We are using Ruby `3.3.0` and Rails `7.1.3.2`.

I assume this issue was caused by [this new file](https://github.com/pboling/sanitize_email/compare/v2.0.4...v2.0.5#diff-6b14c6f0901cbec9229126c0ae73f28b72421a354fc0aead148fa406f5f665ca) on v2.0.5, but I didn't dig deeper to investigate it.

LMK if there's anything else I can provide to help you debug/investigate this issue. :pray:

### Logs

```
-----> Preparing app for Rails asset pipeline
Running: rake assets:precompile
rake aborted!

NameError: undefined local variable or method `config' for an instance of #<Class:0x00007f2680aad400> (NameError)

     config.action_mailer.register_interceptor(SanitizeEmail::Bleach)
     ^^^^^^

/vendor/bundle/ruby/3.3.0/gems/sanitize_email-2.0.5/lib/sanitize_email/engine_v6.rb:12:in `block in <class:EngineV5>'
/vendor/bundle/ruby/3.3.0/gems/activesupport-7.1.3.2/lib/active_support/callbacks.rb:448:in `instance_exec'
/vendor/bundle/ruby/3.3.0/gems/activesupport-7.1.3.2/lib/active_support/callbacks.rb:448:in `block in make_lambda'
/vendor/bundle/ruby/3.3.0/gems/activesupport-7.1.3.2/lib/active_support/callbacks.rb:202:in `block (2 levels) in halting'
/vendor/bundle/ruby/3.3.0/gems/activesupport-7.1.3.2/lib/active_support/callbacks.rb:707:in `block (2 levels) in default_terminator'
/vendor/bundle/ruby/3.3.0/gems/activesupport-7.1.3.2/lib/active_support/callbacks.rb:706:in `catch'
/vendor/bundle/ruby/3.3.0/gems/activesupport-7.1.3.2/lib/active_support/callbacks.rb:706:in `block in default_terminator'
/vendor/bundle/ruby/3.3.0/gems/activesupport-7.1.3.2/lib/active_support/callbacks.rb:203:in `block in halting'
/vendor/bundle/ruby/3.3.0/gems/activesupport-7.1.3.2/lib/active_support/callbacks.rb:598:in `block in invoke_before'
/vendor/bundle/ruby/3.3.0/gems/activesupport-7.1.3.2/lib/active_support/callbacks.rb:598:in `each'
/vendor/bundle/ruby/3.3.0/gems/activesupport-7.1.3.2/lib/active_support/callbacks.rb:598:in `invoke_before'
/vendor/bundle/ruby/3.3.0/gems/activesupport-7.1.3.2/lib/active_support/callbacks.rb:109:in `run_callbacks'
/vendor/bundle/ruby/3.3.0/gems/activesupport-7.1.3.2/lib/active_support/reloader.rb:96:in `prepare!'
/vendor/bundle/ruby/3.3.0/gems/railties-7.1.3.2/lib/rails/application/finisher.rb:74:in `block in <module:Finisher>'
/vendor/bundle/ruby/3.3.0/gems/railties-7.1.3.2/lib/rails/initializable.rb:32:in `instance_exec'
/vendor/bundle/ruby/3.3.0/gems/railties-7.1.3.2/lib/rails/initializable.rb:32:in `run'
/vendor/bundle/ruby/3.3.0/gems/railties-7.1.3.2/lib/rails/initializable.rb:61:in `block in run_initializers'
/vendor/bundle/ruby/3.3.0/gems/railties-7.1.3.2/lib/rails/initializable.rb:60:in `run_initializers'
/vendor/bundle/ruby/3.3.0/gems/railties-7.1.3.2/lib/rails/application.rb:426:in `initialize!'
/config/environment.rb:7:in `<main>'
/vendor/bundle/ruby/3.3.0/gems/bootsnap-1.18.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
/vendor/bundle/ruby/3.3.0/gems/zeitwerk-2.6.13/lib/zeitwerk/kernel.rb:34:in `require'
/vendor/bundle/ruby/3.3.0/gems/railties-7.1.3.2/lib/rails/application.rb:402:in `require_environment!'
/vendor/bundle/ruby/3.3.0/gems/railties-7.1.3.2/lib/rails/application.rb:588:in `block in run_tasks_blocks'
/Rakefile:19:in `invoke'
/vendor/bundle/ruby/3.3.0/gems/sprockets-rails-3.4.2/lib/sprockets/rails/task.rb:61:in `block (2 levels) in define'
/Rakefile:14:in `invoke'
Tasks: TOP => environment
(See full trace by running task with --trace)
```

**Comment by pboling** [comment]
@TiuTalk Thanks!  I'll get it fixed!

**Comment by pboling** [comment]
Fixes are done, and 100% tested in the shiny new combustion + appraisal GHA test matrix - please check it if you get a chance!  Release v2.0.6 coming soon.

**Comment by TiuTalk** [comment]
Awesome, thank you! 🙏

**Comment by pboling** [comment]
There was a problem in the engine for Rails 6+ in the 2.0.6 release.  Fixed in 2.0.7 @TiuTalk
